src/lsst/cmservice/cli/server.py

```diff
@@ -4,13 +4,13 @@
 from safir.asyncio import run_with_asyncio
 from safir.database import create_database_engine, initialize_database
 
-from .. import db
+from .. import __version__, db
 from ..config import config
 
 
 # build the server CLI
 @click.group()
-@click.version_option(package_name="lsst-cm-service")
+@click.version_option(version=__version__)
```

Comment:
Does `__version__` somehow magically get set to git-tag-based versioning as is done with the `package_name` call?  Coz' that was pretty useful I thought...